Add QuickAction node

rathga opened the pull request:
I am writing a flow to combine Life360 events (i.e. leaving my house) with starting and stopping the evohome 'Away' quick action, so I have added a QuickAction node to implement this.  Let me know if it looks good.

dehsgr reviewed (changes requested):
Well done, @rathga! I really appreciate your work on this repository. I would like you to reflect the following changes: 
- README.md, Line 32: End your sentence with a dot directly followed by the following sentence.
- evohome-control.js: revert (unintended?) indentations/disindentation.
- evohome-quickaction.html, Line 8: check comment indentation; Line 26-27+34: check code indentation.
- evohome-quickactions.js: change line 20+21 to verbose logging only; remove "Evohome failed:"-fragment from lines 23+26.
- package.json: I suggest to raise version to v1.2.0 as this version brings a new feature.

Thank you very, very much again. After performing your changes I'll appreciate to merge your work. 

All the best Greetings!

rathga commented:
Thanks @dehsgr !

All done except:

- evohome-control.js: fixing up the indentation was intentional but I see my editor has spammed spaces instead of tabs, so I've fixed that bit
- evohome-quickaction.js: line 21 node.log(...) seems consistent log level with evohome-control.js?  As does the "evohome failed" warnings?  Have I misunderstood? I have updated line 20 to debug.

Let me know if you agree with those, happy either way.

Cheers
Richard

dehsgr commented:
Thanks again, @rathga. I finally merge and publish a new version ASAP.
